py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py

```diff
@@ -282,9 +282,16 @@ def _populate_trt_builder_config(
         if self.compilation_settings.disable_tf32:
             builder_config.clear_flag(trt.BuilderFlag.TF32)
 
-        if self.compilation_settings.make_refittable:
+        if version.parse(trt.__version__) >= version.parse("10.0"):
+            if self.compilation_settings.refit_identical_engine_weights:
+                builder_config.set_flag(trt.BuilderFlag.REFIT_IDENTICAL)
+            else:
+                builder_config.set_flag(trt.BuilderFlag.REFIT)
+        else:
             builder_config.set_flag(trt.BuilderFlag.REFIT)
 
+        builder_config.set_flag(trt.BuilderFlag.STRIP_PLAN)
```

Comment:
It looks like you were reviewing an outdated commit (not submit after that review lol). To clarify the intention here, previously we planned to embed weight stripping into the compilation workflow whatever `strip_engine_weights` is because it would benefit engine caching. Afterwards, we will add weights back if users want a weighted engine. However, due to the lack of TRT support, as we discussed, it was deprecated. So for now the latest commit added the if condition.